gsoc/2020/mentors.md

```diff
@@ -5,7 +5,10 @@ layout: plain
 
 ## Full Mentor List (Name, Email, Org)
 
```

Comment:
After resolving the remaining conversations, please rebase against the current master and fix conflicts in mentors.md file, including the ordering